config/osu.php

```diff
@@ -312,4 +312,8 @@
         'country_performance_user_count' => intval(env('COUNTRY_PERFORMANCE_USER_COUNT', 1000)),
         'country_performance_weighting_factor' => floatval(env('COUNTRY_PERFORMANCE_WEIGHTING_FACTOR', 0.99)),
     ],
+    'tags' => [
+        'tags_cache_interval' => 60 * intval(env('TAGS_CACHE_INTERVAL', 60)), // in minutes, converted to seconds
+        'beatmap_tags_cache_interval' => 60 * intval(env('BEATMAP_TAGS_CACHE_INTERVAL', 60)), // in minutes, converted to seconds
```

Comment:
We're standardizing on `_duration` suffix for cache durations and the keys should be alphabetically ordered (I just noticed the existing keys at the bottom are in the wrong place but that's a different problem).

Also, use `get_int(env(ENVKEY)) ?? 60` instead of `intval`.